training/scaleway_gpu.py

```
@@ -0,0 +1,600 @@
+		------
+		RuntimeError
+			If instance is not running or cannot retrieve IP address.
+		"""
+		import subprocess
+		import sys
+		import threading
+
+		# Ensure instance is running
+		state = self.status()
+		if state != "running":
+			raise RuntimeError(f"Instance must be running to execute commands (current state: {state})")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)